### PR TITLE
MAISTRA-2132 Support deployment of multiple plugin versions in Istio CNI

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

**Review by brian-avery — APPROVED** [review]

File: cni/pkg/install-cni/cmd/root.go [hunk 1]
```diff
@@ -138,6 +138,7 @@ func constructConfig() (*config.Config, error) {
 
 		UpdateCNIBinaries: viper.GetBool(constants.UpdateCNIBinaries),
 		SkipCNIBinaries:   viper.GetStringSlice(constants.SkipCNIBinaries),
+		CNIBinariesPrefix: viper.GetString(constants.CNIBinariesPrefix),
```
Comment (jwendell) [inline cni/pkg/install-cni/cmd/root.go:142]: I think you need to register it in `init()`

**Review by jwendell — COMMENTED** [review]
Worth to add unit tests? And submit upstream after it?

File: cni/pkg/install-cni/cmd/root.go (hunk 1 above)
Comment (luksa) [reply cni/pkg/install-cni/cmd/root.go:142]: Thanks! Fixed.

**Review by jwendell — APPROVED** [review]
We can merge this as is and add unit tests in a separate PR: https://issues.redhat.com/browse/MAISTRA-2135

**Pull request merged**